src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -206,6 +207,54 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     fromAddress: fromAddress ?? null,
   })
 
+  const { amountCryptoBaseUnitMinusFees, amountCryptoHumanMinusFees } = useMemo(() => {
+    if (!estimatedFeesData)
+      return {
+        amountCryptoBaseUnitMinusFees: undefined,
+        amountCryptoHumanMinusFees: undefined,
+      }
+
+    if (
+      state?.deposit.sendMax &&
+      estimatedFeesData &&
+      !isUtxoChainId(feeAsset.chainId) &&
+      feeAsset.assetId === asset.assetId
+    ) {
+      return {
+        amountCryptoBaseUnitMinusFees: bn(
+          toBaseUnit(state?.deposit.cryptoAmount, asset.precision),
+        ).minus(estimatedFeesData.txFeeCryptoBaseUnit),
+        amountCryptoHumanMinusFees: fromBaseUnit(
+          bn(toBaseUnit(state?.deposit.cryptoAmount, asset.precision)).minus(
+            estimatedFeesData.txFeeCryptoBaseUnit,
+          ),
+          asset.precision,
+        ),
+      }
+    }
+
+    return {
+      amountCryptoBaseUnitMinusFees: bn(toBaseUnit(state?.deposit.cryptoAmount, asset.precision)),
+      amountCryptoHumanMinusFees: state?.deposit.cryptoAmount,
+    }
+  }, [
+    state?.deposit.cryptoAmount,
+    asset.precision,
+    feeAsset,
+    asset.assetId,
+    estimatedFeesData,
+    state?.deposit.sendMax,
+  ])
+
+  const { executeTransaction } = useSendThorTx({
+    accountId: accountId ?? null,
+    assetId,
+    amountCryptoBaseUnit: amountCryptoBaseUnitMinusFees?.toFixed() ?? null,
+    action: isRunePool ? 'depositRunepool' : 'depositSavers',
+    memo,
+    fromAddress: fromAddress ?? null,
```

Comment:
preferably-blocking: disable additional fees estimation reqs for the flows where we're not concerned about fees deduction (i.e non send maxes, non-UTXOs, non-native assets). We already estimate fees in the `useSendThorTx()` call above

```suggestion
    fromAddress: fromAddress ?? null,
    enableEstimateFees: Boolean(
      !state?.deposit.sendMax ||
      isUtxoChainId(feeAsset.chainId) ||
      feeAsset.assetId !== asset.assetId
    ) 
```